### PR TITLE
fix display

### PR DESCRIPTION
**Pull request by PiranhaSa**
image
image

### CODE REVIEW

File: WEB_SERVICE/WEB/_HARDENING/WEB_HTTP_CSP/description.md [hunk 1]
```diff
@@ -1,57 +1,47 @@
-Content Security Policy (CSP) is a computer security standard that provides an added layer of protection against
-Cross-Site Scripting (XSS), clickjacking, and other client-side attacks that rely on executing malicious content in the
-context of a trusted web resource.
+**Content Security Policy** (CSP): is a computer security standard that provides an added layer of protection against Cross-Site Scripting (XSS), clickjacking, and other client-side attacks that rely on executing malicious content in the context of a trusted web resource.
```
Comment (Zakaria-Latif) [inline WEB_SERVICE/WEB/_HARDENING/WEB_HTTP_CSP/description.md:1]:
```suggestion
**Content Security Policy** (CSP): A computer security standard that provides an added layer of protection against Cross-Site Scripting (XSS), clickjacking, and other client-side attacks that rely on executing malicious content in the context of a trusted web resource.
```

File: WEB_SERVICE/WEB/_HARDENING/WEB_HTTP_CSP/description.md [hunk 2]
```diff
@@ -1,57 +1,47 @@
...
 
-CSP can be enabled by instructing the browser with a Content-Security-Policy directive in a response header;
+CSP can be enabled by instructing the browser with a Content-Security-Policy directive in a response header:
 
 ```html
  Content-Security-Policy: script-src 'self';
 ```
 
-or in a meta tag;
+Or in a meta tag;
```
Comment (Zakaria-Latif) [inline WEB_SERVICE/WEB/_HARDENING/WEB_HTTP_CSP/description.md:11]:
```suggestion
Or in a meta tag:
```

File: WEB_SERVICE/WEB/_HARDENING/WEB_HTTP_CSP/description.md [hunk 3]
```diff
@@ -1,57 +1,47 @@
...
-    - object-src
-    - script-src
-    - style-src
-      When setting the CSP directives, you can also use some CSP keywords:
-
-- none: Denies loading resources from anywhere.
-- self : Points to the document's URL (domain + port).
-- unsafe-inline: Permits running inline scripts.
-- unsafe-eval: Permits execution of evaluation functions such as eval().
+You can restrict script loading only to the same domain in the above example. It will also limit inline script executions in the element attributes and the event handlers.
```
Comment (Zakaria-Latif) [inline WEB_SERVICE/WEB/_HARDENING/WEB_HTTP_CSP/description.md:17]:
```suggestion
You can restrict script loading only to the same domain in the above example. It also limits inline script execution in element attributes and event handlers. 
```

File: WEB_SERVICE/WEB/_HARDENING/WEB_HTTP_CSP/description.md [hunk 4]
```diff
@@ -1,57 +1,47 @@
...
+You can restrict script loading only to the same domain in the above example. It will also limit inline script executions in the element attributes and the event handlers. 
+
+There are various directives that you can use by declaring CSP:
+
+- **script-src**: Restricts the script loading resources to the ones you declared. By default, it disables inline script executions unless you permit the evaluation functions and inline scripts by the unsafe-eval and unsafe-inline keywords.
+- **base-uri**: The base element is used to resolve a relative URL to an absolute one. By using this CSP directive, you can define all possible URLs which could be assigned to the base-href attribute of the document. 
+- **frame-ancestors**: It is very similar to X-Frame-Options HTTP header. It defines the URLs by which the page can be loaded in an iframe.
+- **frame-src / child-src**: frame-src is the deprecated version of child-src. Both define the sources that can be loaded by iframe on the page. (Please note that frame-src was brought back in CSP 3)
+- **object-src** : Defines the resources that can be loaded by embedding such as Flash files, Java Applets.
```
Comment (Zakaria-Latif) [inline WEB_SERVICE/WEB/_HARDENING/WEB_HTTP_CSP/description.md:25]:
```suggestion
- **object-src**: Defines the resources that can be loaded by embedding such as Flash files, Java Applets.
```

**Comment by mohsinenar** [comment]
@PiranhaSa Please update the title!

**Review by mohsinenar — DISMISSED** [review]
Please update the title and descreption of the PR and explain what changed!

File: WEB_SERVICE/WEB/_HARDENING/WEB_HTTP_CSP/description.md (hunk 1 above)
Comment (benyissa) [reply WEB_SERVICE/WEB/_HARDENING/WEB_HTTP_CSP/description.md:1]: @Zakaria-Latif, I think this is out of the PR context.

**Review by 3asm — APPROVED** [review]

File: WEB_SERVICE/WEB/_HARDENING/WEB_HTTP_CSP/description.md (hunk 1 above)
Comment (Zakaria-Latif) [reply WEB_SERVICE/WEB/_HARDENING/WEB_HTTP_CSP/description.md:1]: The PR says `fix display`, I thought it was meant to correct typos, grammar, and spelling !!.

**Pull request merged**